doctor/cores/text_processor.py

```
@@ -0,0 +1,462 @@
+    sentiment: str
+    language: str
+    medical_relevance: float
+
+
+class DoctorTextProcessorCore:
+    """
+    هسته پردازش متن برای اپ Doctor
+    مسئول پردازش نوت‌های پزشکی، استخراج اطلاعات و تولید گزارش‌ها
+    """
+    
+    def __init__(self):
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)